### ISSUE TITLE
Make SteemHunt Fun Again

### ISSUE THREAD

**Issue by ThaBiGGDoGG** [Steemhunt/web#187]
SteemHunt has been updating many things. It is hard to please everyone. I have a idea that might make it more fun. We currently are limited to 2 Hunts per day. What if we could have a way to unlock 1 or 2 more hunts each per day? Now this is where the FUN Part comes in. You would spent some of your HUNT Tokens to purchase the additional extra hunt(s) that day.

Token cost can be determined by Dev team. Those Tokens used to purchase an addition Hunt could be put into collection & pay out to everyone as a bonus weekly. That could be decided how that would work. 

Also some people do not like the new APV. What if you could spend some HUNT Tokens to increase your Hunt Score? It would be limited to a max amount. It would not allow a person in last place to jump to first. But it might give them a small boost. Again those tokens spent would go to everyone Weekly as a Bonus.

These are just 2 ideas on a way to add HUNT Token Spending into the platform. And make it a little more fun.

**Comment by Chuuuckie** [comment]
The problem here is that we cannot trade our tokens YET.

**Comment by ThaBiGGDoGG** [comment]
Yes, but this could be the next step after APV. When token trading goes live.

**Comment by Chuuuckie** [comment]
Its a good idea tho.

**Comment by sydneyitguy** [comment]
interesting idea :)

we'll think about it as a long-term (currently too many stuff's on higher priority)

**Comment by fknmayhem** [comment]
Hunt deathmatches. 

My hunt against your hunt. Buy in with (capped amount of HUNT), two enter the HUNTdome. Winner takes all (90% -10% for Steemhunt). 

One day each week. Save your best hunts for DeathHUNT!

Yes, long term LOL.

**Comment by ThaBiGGDoGG** [comment]
DeathHUNT could be weekly. It could add up your top 3 or 5 hunts for the week for a score. That way if one person had a bunch go vote for a hunt, you couldn't win with just that.

**Comment by fknmayhem** [comment]
Meh, weekly would only result in those who already received most getting even more.

Not a problem if you can gather more upvoters to vote for you. Just hope that in the short window you’re in the HUNTdome there’s not even more, or less but with a higher diversity score, who vote for the other project.

Enter, and spend the entry fee, at your own risk.
